transactionoutbox-acceptance/src/test/java/com/gruelbox/transactionoutbox/acceptance/persistor/TestDefaultPersistorMSSqlServer2017.java

```diff
@@ -0,0 +1,47 @@
+package com.gruelbox.transactionoutbox.acceptance.persistor;
```

Comment:
Are all these SQL server implementations different enough to justify the tests?

Don't mind really, just worth checking

Reply:
It is a good question. The SQL used should be supported by all versions of the database. This was my way of testing that that was true and that hopefully any change, either in the library or in the databases, would be caught. I don't have a problem removing the other instances, though, just let me know your preference.